applications/nrf5340_audio/src/modules/sd_card_playback.c

```diff
@@ -0,0 +1,402 @@
+#include "sd_card_playback.h"
+#include "sd_card.h"
+#include "sw_codec_lc3.h"
+#include "sw_codec_select.h"
+#include "pcm_mix.h"
+#include <stdint.h>
+#include <zephyr/sys/ring_buffer.h>
+#include <zephyr/shell/shell.h>
```

Comment:
I had a discussion with Andreas and Kristoffer about the fact that the module can only play when the BT stream is up and running. We concluded that this could be fixed in the future since it would require a lot of extra work